core/src/crds_gossip_pull.rs

```diff
@@ -215,6 +219,11 @@ impl CrdsGossipPull {
     ) -> usize {
         let mut failed = 0;
         for r in response {
+            if now > r.wallclock() + self.msg_timeout || now + self.msg_timeout < r.wallclock() {
+                inc_new_counter_error!("cluster_info-gossip_pull_response_value_timeout", 1);
```

Comment:
should we have a different counter for older and newer timeouts?

Reply:
Hm newer is only going to help spot liars. I followed what the push message logic does. There it returns the same error for old and new.